articles/hooks/guides/post-change-password.md

```diff
@@ -0,0 +1,131 @@
+---
+description: Learn about the Post Change Password Hook available for Database Connections.
+beta: true
+topics:
+    - hooks
+    - extensibility-points
+    - post-password-change
+contentType:
+    - how-to
+useCase: extensibility-hooks
+v2: true
+---
+
+# Post Change Password Hook
+
+The Post Change Password Hook allows custom actions to be executed after a successful user password change; when a user updates their password or after a tenant administrator updates a user's password. This hook executes asynchronously with the rest of the Auth0 pipeline and its outcome does not affect the Auth0 transaction.
+
+With the Post Change Password Hook, you can do things like:
+
+* Send an email to a user letting them know their password has been changed.
+* Logout a user so they have to login with their new password.
+
+The Post Change Password Hook is only available for [Database Connections](/connections/database).
+
+You can create a new Post Change Password hook using the [Dashboard](/hooks/guides/create-hooks-using-dashboard) or the [Command Line Interface](/hooks/guides/create-hooks-using-cli).
+
+## Starter code and parameters
+
+After you've created a new Post Change Password Hook, open up the Hook and edit it using the Webtask Editor embedded in the Dashboard. 
+
+The parameters listed in the comment at the top of the code indicate the Auth0 objects and their parameters that can be passed into and used by the Hook's function.
+
+```js
+/**
+@param {object} user - The affected user
+@param {string} user.id - user id
+@param {string} user.username - user name
+@param {string} user.email - email
+@param {string} user.last_password_reset - exact date/time the user's password was changed
+@param {object} context - Auth0 connection and other context info
+@param {object} context.connection - information about the Auth0 connection
+@param {object} context.connection.id - connection id
+@param {object} context.connection.name - connection name
+@param {object} context.connection.tenant - connection tenant
+@param {object} context.webtask - webtask context
+@param {function} cb - function (error, response)
+**/
+module.exports = function (user, context, cb) {
+// Perform any asynchronous actions, e.g. send notification to Slack.
+cb();
+};
+```
+
+The callback function `cb` at the end of the sample code is used to signal completion and must not be omitted.
+
+### Response
+
+The Post Change Password Hook ignores any response object.
```

Comment:
if an error is returned, it does not affect the Auth0 transaction but a tenant log is generated... can we include this?